PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -268,11 +259,51 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
     return rejectionMask;
   }
 
+  /// Get the occupancy
+  /// \param collision is the collision with the occupancy information
+  /// \param occEstimator is the occupancy estimator (1: ITS, 2: FT0C)
+  template <typename Coll>
+  float getOccupancy(Coll const& collision, int occEstimator = 1)
+  {
+    switch (occEstimator) {
+      case 1: // ITS
+        return collision.trackOccupancyInTimeRange();
+        break;
+      case 2: // FT0c
+        return collision.ft0cOccupancyInTimeRange();
+        break;
+      default:
+        LOG(warning) << "Occupancy estimator not valid. Possible values are ITS or FT0C. Fallback to ITS";
+        return collision.trackOccupancyInTimeRange();
+        break;
+    }
```

Comment:
Returning a fallback value just with a warning when an invalid value is provided look like a bad idea to me. Invalid input should cause a fatal error.